### PR TITLE
rand() function

### PR DESCRIPTION
**Pull request by KushagraGoel**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

Fixes #[Add issue number here.]

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: .rubocop_shopify_styleguide.yml [hunk 1]
```diff
@@ -970,8 +970,7 @@ Lint/PercentSymbolArray:
   Enabled: true
 
 Lint/RandOne:
-  Description: Checks for `rand(1)` calls. Such calls always return `0` and most
-    likely a mistake.
+  Description: Checks for `rand(1)` calls.  Such calls always return `0` and most likely a mistake.rand() function is used in C to              generate random numbers .
```
Comment (gauravano) [inline .rubocop_shopify_styleguide.yml]: @jywarren could you tell me why we are having this file?

**Comment by SidharthBansal** [comment]
@jywarren do we need to merge these changes or close this PR?
@KushagraGoel is inactive from 2 months.

**Comment by jywarren** [comment]
Hmm, this adds a bit of extra info, so i guess we can merge it.

**Comment by jywarren** [comment]
Hmm, I'm not sure this is helpful, unfortunately -- I'm going to close it, but we'd love to work with you on another issue, @KushagraGoel ! Thank you!!!!

**Pull request closed without merging**